src/korean.js

```diff
@@ -104,7 +104,7 @@ class Korean {
 
   doPast(word) {
     let presentWord = doPresent(word);
-    // stuff for past tense
+    return presentWord.concat('ㅆ어')
```

Comment:
Right, the basic steps that @cecilebertin mentioned are outline in the issue #14 . Thanks!